docs/make.jl

```diff
@@ -81,4 +81,21 @@ makedocs(
     ],
 )
 
-deploydocs(repo = "github.com/JuliaManifolds/Manifolds.jl.git", push_preview = true)
+function should_push_preview(event_path = get(ENV, "GITHUB_EVENT_PATH", nothing))
+    event_path === nothing && return false
+    event = JSON.parsefile(event_path)
+    labels = [x["name"] for x in event["pull_request"]["labels"]]
+    # https://developer.github.com/v3/activity/events/types/#pullrequestevent
+    yes = "deploydocs" in labels
+    if yes
+        @info "Trying to push preview as label `deploydocs` is specified." labels
+    else
+        @info "Not pushing preview as label `deploydocs` is not specified." labels
+    end
+    return yes
+end
+
+deploydocs(
```

Comment:
One could even speed this up and avoid deploying by adding ` should_push_preview() &&` upfront this line; what do you think?

Reply:
If it's not deploying, I doubt calling `deploydocs` with `push_preview=false` takes much time at all.

Reply:
You're right. Anyways with your other remark and now an approach to a pure GitHub Actions approach this point is not necessary anymore :)